src/main/resources/application.yml

```
@@ -0,0 +1,19 @@
+spring:
+  datasource:
+    url: jdbc:h2:tcp://localhost/~/cafe
```

```suggestion
    url: jdbc:h2:mem://localhost/~/cafe
```
기존 코드는 에러로 어플리케이션 구동이 안되네요!
mem 를 이용해서 메모리로 띄우니까 돌아가네요!